packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,20 +8,33 @@ showTitle: true
 PatternFly offers two components for displaying large data sets: [data list](/documentation/react/components/datalist/) and [table](/documentation/react/components/table/). While they satisfy similar use cases, choosing the correct component to use in your design will be dependent on the type of data you need to display.
 
 Use [data lists](#data-lists) when:
-* A flexible layout is more important than arranging information in a grid
-* You want to include active content like a chart
-* Content displayed may vary between rows
+* A flexible layout is more important than arranging information in a grid.
+* You want to include active content like a chart.
+* Content displayed may vary between rows.
 
 Use [tables](#tables) when:
-* Users will want to consume data as a grid (i.e. structured rows and columns)
-* You want column headers
+* Users will want to consume data as a grid (i.e. structured rows and columns).
+* You want column headers.
 
 ## Data lists
 PatternFly supports several variations of the data list component.
 
-### Standard data list
+### How to use
+Think of each row in a data list as a container for some formatted content. In PatternFly 4, data list rows can accept any [layout](/documentation/react/layouts/bullseye) supported by the design system as long as all rows apply the same layout. Here are some common layouts that may be useful in a data list:
+
+* **[Grid](/documentation/react/layouts/grid):** When you want to display content in a responsive grid.
+* **[Level](/documentation/react/layouts/level):** When you want to justify content evenly over the width of the row.
+* **[Split](/documentation/react/layouts/split):** When you want to distribute content evenly with a main content area in the center.
+
+### Data list capabilities
+Every [compact](#when-to-use-compact-vs-basic-spacing) or [basic](#when-to-use-compact-vs-basic-spacing) data list can be extended with these functionalities.
+
+* **[Actionable data list](#actionable-data-list)**
+* **[Expandable data list](#expandable-data-list)**
```

Comment:
Also you have compact, actionable, and expandable under this heading - should we have an entry here for `[Compact data list](#compact-data-list)`?